source/install/mattermost-omnibus.rst

```diff
@@ -65,13 +65,23 @@ The properties that you can configure in this file are:
 - ``enable_plugin_uploads``: This setting can be ``true`` or ``false`` and is used to configure the ``PluginSettings.EnableUploads`` Mattermost configuration property.
 - ``enable_local_mode``: This setting can be ``true`` or ``false`` and is used to configure the ``ServiceSettings.EnableLocalMode`` Mattermost configuration property.
 - ``nginx_template``: Optional path to a custom NGINX template.
+- ``monitoring_installed``: This setting can be ``true`` or ``false`` and is used to install the monitoring component.
+- ``monitoring_enabled``: This setting can be ``true`` or ``false`` and is used to enable the monitoring component.
+- ``grafana_user``: The monitoring component Grafana username. This setting should be set if the monitoring component is enabled.
+- ``grafana_password``: The monitoring component Grafana password. This setting should be set if the monitoring component is enabled.
+- ``jitsi_installed``: This setting can be ``true`` or ``false`` and is used to install the Jitsi component.
+- ``jitsi_enabled``: This setting can be ``true`` or ``false`` and is used to enable the Jitsi component.
+- ``jitsi_fqdn``: The domain name of the Jitsi component.
+- ``jitsi_jvb_secret``: An internal Jitsi secret used to communicate between its prosody and video-bridge modules.
+- ``jitsi_focus_secret``: An internal Jitsi secret used to communicate between its prosody and jicofo modules.
+- ``jitsi_focus_password``: An internal Jitsi secret used to connect jicofo and prosody.
```

Comment:
Maybe we can rewrite it in a more explanatory way, something like `The password for the "focus" user in prosody. This is used to connect jicofo to prosody.`

Reply:
Addressed in 140e755